ACKategoriesCore/RandomExtensions/Randomizable.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+/// Mark a type Randomizable
+/// to get automatically synthesized `.random()` method for `Collection`s
+public protocol Randomizable {
+    /// - Returns: Random instance of `Self`
+    static func random() -> Self
+}
+
+public extension Collection where Element: Randomizable {
+    /// - Parameters:
+    ///     - min: Minimal number of returned elements
+    ///     - max: Maximal number of returned elements
+    /// - Returns: Array of random elements
+    static func random(min: Int = 20, max: Int = 100) -> [Element] {
+        assert(min <= max, "Min must be smaller than max")
```

Comment:
Assertion and the message say two different things. You could 
- say something like "Min must not be greater than max", 
- say something like "Min must be smaller than or equal to max", 
- require that min is strictly lower than max

Reply:
Deleted the assert altogether, as all the edge cases will be caught in `Int.random(in: min...max)`